src/app/presentation/components/home-components/about/about.component.html

```diff
@@ -63,10 +63,10 @@ <h3 class="feature-title">About our product</h3>
     <section class="about-top-section bottom-section d-flex">
         <div class="about-section">
           <div class="titled">
-              <span>Some title</span>
+              <span>Let's start</span>
             </div>
             <div class="bottom-section-text">
-              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+              <p>We hope that you are ready for start the new era in yours buisnes. Because after using Provis, you will can't return to your past way of managing. Good luck!</p>
```

Comment:
We hope that you are ready to start the new era of your business. After using Provis, you won't be able to return to your previous way of managing. Good luck!